media-source-respec.html

```diff
@@ -599,6 +688,40 @@ <h3>Event Summary</h3>
         </table>
       </section>
 
+      <section id="mediasource-in-worker-communication-model">
+        <h3><dfn>Cross-context communication model</dfn></h3>
+
+        <p>When a {{Window}} {{HTMLMediaElement}} is attached to a {{DedicatedWorkerGlobalScope}} {{MediaSource}}, each context
+        has algorithms that depend on information from the other.</p>
+
+        <p class="note">{{HTMLMediaElement}} is exposed only to {{Window}} contexts, but {{MediaSource}} and related objects defined
+        in this specification are exposed in {{Window}} and {{DedicatedWorkerGlobalScope}} contexts. This lets
+        applications construct a {{MediaSource}} object in either of those types of context and attach it to an
+        {{HTMLMediaElement}} object in a {{Window}} context using a [=MediaSource object URL=] as described in the
+        [=attaching to a media element=] algorithm. A {{MediaSource}} object is not {{Transferable}}; it is only visible
+        in the context where it was created.</p>
+
+        The rest of this section describes a model for bounding information latency for attachments of a {{Window}}
+        media element to a {{DedicatedWorkerGlobalScope}} {{MediaSource}}. While the model describes communication using
+        message passing, implementations MAY choose to communicate in potentially faster ways, such as using shared
+        memory and locks. Attachments to a {{Window}} {{MediaSource}} synchronously have the information already without
+        communicating it across contexts.
+
+        <p>A {{MediaSource}} that is constructed in a {{DedicatedWorkerGlobalScope}} has an internal, unexposed
+        <dfn id="port-to-main">port to main</dfn> nullable variable that stores a {{MessagePort}} setup during
+        attachment and nulled during detachment. A {{Window}} {{MediaSource}}'s [=port to main=] is always null.</p>
+
+        </p>An {{HTMLMediaElement}} extended by this specification and attached to a {{DedicatedWorkerGlobalScope}}
+        {{MediaSource}} similarly has an internal, unexposed <dfn id="port-to-worker">port to worker</dfn> nullable
+        variable that stores a {{MessagePort}} setup during attachment and nulled during detachment. A
+        [=port to worker=] is always null unless attached to a {{DedicatedWorkerGlobalScope}} {{MediaSource}}.</p>
+
```

Comment:
I would also add the definition of the `channel` internal slot here as well, even though it is only going to be set in objects created in dedicated workers.

Reply:
I agree and will update.

Reply:
98fe6fc should fix this.